frontend/packages/kubevirt-plugin/src/statuses/v2v/v2vvmware-status.ts

```diff
@@ -1,6 +1,7 @@
 import { K8sResourceKind } from '@console/internal/module/k8s';
 import { getStatusPhase } from '../../selectors/selectors';
 import { V2VProviderStatus } from './constants';
+import { OVirtProviderModel } from '../../models';
 
 type V2VProviderStatusFlags = {
   hasConnectionFailed?: boolean;
```

Comment:
can we rename the `hasV2vVMWareStatus` method?

Reply:
Yes! Done. =)